### PR TITLE
Parsers Juniper

### PR DESCRIPTION
**Pull request by alxarch**
## Background

Backport Juniper parsers to OSS

## Changes

- Adds parser for `Juniper.Access`
- Adds parser for `Juniper.MWS`
- Adds parser for `Juniper.Security`
- Adds parser for `Juniper.Firewall`
- Adds parser for `Juniper.Audit`
- Adds parser for `Juniper.Postgres`

## Testing

- mage test:ci
- mage deploy && e2e test

### CODE REVIEW

File: internal/log_analysis/log_processor/registry/registry.go [hunk 1]
```diff
@@ -18,6 +18,15 @@ package registry
...
 
+/**
+ * Copyright (C) 2020 Panther Labs Inc
+ *
+ * Panther Enterprise is licensed under the terms of a commercial license available from
+ * Panther Labs Inc ("Panther Commercial License") by contacting contact@runpanther.com.
+ * All use, distribution, and/or modification of this software, whether commercial or non-commercial,
+ * falls under the Panther Commercial License to the extent it is permitted.
+ */
+
```
Comment (kostaspap) [inline internal/log_analysis/log_processor/registry/registry.go:29]: Remove

File: internal/log_analysis/log_processor/parsers/juniperlogs/security_test.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+ */
+
+/**
+ * Copyright (C) 2020 Panther Labs Inc
+ *
+ * Panther Enterprise is licensed under the terms of a commercial license available from
+ * Panther Labs Inc ("Panther Commercial License") by contacting contact@runpanther.com.
+ * All use, distribution, and/or modification of this software, whether commercial or non-commercial,
+ * falls under the Panther Commercial License to the extent it is permitted.
+ */
```
Comment (kostaspap) [inline internal/log_analysis/log_processor/parsers/juniperlogs/security_test.go:28]: Remove

File: internal/log_analysis/log_processor/parsers/juniperlogs/access.go [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+ */
+
+/**
+ * Copyright (C) 2020 Panther Labs Inc
+ *
+ * Panther Enterprise is licensed under the terms of a commercial license available from
+ * Panther Labs Inc ("Panther Commercial License") by contacting contact@runpanther.com.
+ * All use, distribution, and/or modification of this software, whether commercial or non-commercial,
+ * falls under the Panther Commercial License to the extent it is permitted.
+ */
```
Comment (kostaspap) [inline internal/log_analysis/log_processor/parsers/juniperlogs/access.go:28]: Remove

Comment (alxarch) [reply internal/log_analysis/log_processor/parsers/juniperlogs/access.go]: I did a search to locate all duplicates. Ther are some other duplicate license blocks at `internal/core/source_api/api` and `pkg/awssqs`. Should I fix them in this PR?

Comment (kostaspap) [reply internal/log_analysis/log_processor/parsers/juniperlogs/access.go]: Ah yes, sure!

**Pull request merged**